### PR TITLE
Update UG for DeliveryViewCommand and DeliveryCreateNoteCommand

### LINKED ISSUE #309: UG - Delivery Note, Delivery View

### PR DESCRIPTION
**Pull request by B-enguin**
Changed UG formatting of features to be inline with new team styling.

Closes #309.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -528,33 +543,37 @@ Edits Delivery 2's name to Vanilla Cake as it was misspelled and edits the custo
...
 
-**Command succeeds:**
+**Example:** 
 
-![Delivery Note](images/delivery/delivery_note.png)
+* `delivery note 1 --note By FedEx`{.swift} </br>
+  Creates a new Note "By FedEx" for the Delivery with ID `1`{.swift}
 
-**Command failed (invalid_index):** _The delivery index provided is invalid_
+**After:**
```
Comment (zhonghan721) [inline docs/UserGuide.md]: Is this supposed to be `Before`?

**Review by zhonghan721 — COMMENTED** [review]
Some comments, otherwise LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (B-enguin) [reply docs/UserGuide.md]: good catch

File: docs/UserGuide.md [hunk 2]
```diff
@@ -488,23 +488,35 @@ _SORT_: String of either `asc` for ascending or `desc` for descending or default
...
+![Delivery View Before](images/delivery/delivery_view_before.png)
 
-**Command succeeds:**
+**After:**
 
-![](images/delivery/delivery_view.png)
+![Delivery View](images/delivery/delivery_view.png)
 
-**Command failed (0 deliveries):** _There are currently no deliveries._
+[&uarr; Back to Table of Contents](#top)
```
Comment (juliusgambe) [inline docs/UserGuide.md]: Wait, this was done in my PR

**Review by juliusgambe — COMMENTED** [review]
We overallped on DeliveryViewCommand

**Review by juliusgambe — APPROVED** [review]
lgtm

**Pull request merged**